rpkg_src/thirdparty/zhmtools/ResourceLib.h

```diff
@@ -2,6 +2,12 @@
 #include "ResourceGenerator.h"
 #include "ResourceLibCommon.h"
 
+#if __EMSCRIPTEN__
+#include <emscripten.h>
+#else
+#define EMSCRIPTEN_KEEPALIVE
+#endif
```

Comment:
Same here.

Reply:
You could argue the same for HMA. But it's being kept as it's directly from ResourceLib's include folder.